portality/templates/admin/edit_article_metadata.html

```diff
@@ -0,0 +1,161 @@
+{% extends "admin/admin_base.html" %}
+
+{% block content %}
+{% set object_type = 'article' %}
+{% set heading_object_type = 'Article' %}
+{% set form_context = form_context %}
+
+<div class="row" style="margin-bottom: 20px">
+    <div class="col-md-9">
+        <h1>{{ heading_object_type }}: {{form_context.source.bibjson().title}}</h1>
+        {% if form_context.source.last_updated %}
+          <h4>Last updated: {{form_context.source.last_updated}}</h4>
+        {% endif %}
+        {% if form_context.source.current_journal %}
+            <h4 style="margin-top: 1em;"><a href="{{ url_for('doaj.journal_readonly', journal_id=form_context.source.current_journal) }}" target="_blank">View associated journal</a></h4>
+        {% endif %}
+    </div>
+</div>
+
+<div class="row">
+<div class="col-md-12">
+
+    {% if there_were_errors %}
+        <h4 class="red">There is a problem with the submitted form.</h4>
+    {% endif %}
+
+    {% from "_formhelpers.html" import render_field_horizontal %}
+    {% from "_formhelpers.html" import render_field %}
+    <form method="post" action="{{ url_for('admin.article_page', article_id=form.id) }}" class="form-horizontal" id="article_metadata_form">
+
+        {% include "formcontext/maned_article_review.html" %}
+
+    </form>
+        </div>
+            </div>
+
+{% endblock %}
+
+{% block extra_js_bottom %}
+<script type="text/javascript">
+$(document).ready(function() {
```

Comment:
I didn't look in detail, but I presume that the JS here and the JS in the publisher form is the same, so we should put that all in a single file, and use it in both places rather than having copies.

I'd also encourage us to move it to the new JS style, which involves creating or extending a `doaj` global variable and placing all the features in there.